integration-tests/entry-point.sh

```diff
@@ -60,6 +57,24 @@ chmod 640 ${CONFIG_PATH}
 # start ssh
 service ssh start || true
 
+# Ensure lockfile exists
+mkdir -p /run/lock > dev/null
+touch /run/lock/devicecl.lock
```

Comment:
This is redundant because > and >> will both create the file

Reply:
>> appends to the end without overwriting the beginning